src/CMakeLists.txt

```diff
@@ -1062,6 +1062,15 @@ if (UNIX)
         message(STATUS "Linking with CMake DL libs: '${CMAKE_DL_LIBS}'")
         target_link_libraries(${LIBRARY_PROJECT} PUBLIC ${CMAKE_DL_LIBS})
     endif ()
+
+    if (NOT APPLE AND BUILD_WITH_LIBNOTIFY)
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules(LIBNOTIFY REQUIRED libnotify)
+        include_directories(${LIBRARY_PROJECT} PUBLIC ${LIBNOTIFY_INCLUDE_DIRS})
+        target_link_libraries(${LIBRARY_PROJECT} PUBLIC ${LIBNOTIFY_LIBRARIES})
+        target_link_directories(${LIBRARY_PROJECT} PUBLIC ${LIBNOTIFY_LIBRARY_DIRS})
```

Comment:
I'm pretty sure we can link to `libnotify` as `PRIVATE` as we only use it internally (not exposed in any header or through other functions). We should define `CHATTERINO_WITH_LIBNOTIFY` as `PUBLIC` however, as it changes the layout of `Toasts`.

Reply:
If this works that's fine :+1: